### PR TITLE
Restore deployer functionality

### LINKED ISSUE #33: canDeploy modifier
Right now the following line is used in two deployment related functions:
` require(authorizedDeploy[msg.sender]);`

It may be relevant to move that logic into a modifier, specially if we implement `require(condition, message)` in the future.

### LINKED ISSUE #116: LANDRegistry make LANDAuction contract deployer

### PR DESCRIPTION
**Pull request by abarmat**
Closes #116 
Closes #33

### CODE REVIEW

File: contracts/land/LANDStorage.sol [hunk 1]
```diff
@@ -10,9 +10,11 @@ contract LANDStorage {
   uint256 constant clearHigh = 0x00000000000000000000000000000000ffffffffffffffffffffffffffffffff;
   uint256 constant factor = 0x100000000000000000000000000000000;
 
-  mapping (address => bool) public authorizedDeploy;
+  mapping (address => bool) internal _deprecated1;
```
Comment (nachomazzara) [inline contracts/land/LANDStorage.sol]: Maybe we can call it more specific:
`_deprecated_authorizedDeploy`
`_deprecated_authorizedDeploy_11_01_2018` (with a date)

File: contracts/land/LANDRegistry.sol [hunk 2]
```diff
@@ -50,6 +51,10 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     _;
   }
 
+  //
+  // LAND Create and destroy
```
Comment (nachomazzara) [inline contracts/land/LANDRegistry.sol]: I think this comment does not belong here. How can we destroy LANDs?

File: test/LANDRegistry.js [hunk 3]
```diff
@@ -387,6 +387,17 @@ contract('LANDRegistry', accounts => {
         await land.initialize(hacker, { from: hacker })
         await assertRevert(land.forbidDeploy(hacker, { from: hacker }))
       })
+
+      it('throw if user tries to assign LAND and it not deployer', async function() {
```
Comment (nachomazzara) [inline test/LANDRegistry.js]: I think it is better to put `it('reverts....`

File: truffle-config.js [hunk 4]
```diff
@@ -4,6 +4,10 @@ require('babel-polyfill')
 const HDWalletProvider = require('truffle-hdwallet-provider')
 const mnemonic = '' // 12 word mnemonic
```
Comment (eordano) [inline truffle-config.js]: append: `|| process.env('MNEMONIC')`

File: contracts/land/LANDRegistry.sol (hunk 2 above)
Comment (abarmat) [reply contracts/land/LANDRegistry.sol]: Should be removed it's an old comment

File: truffle-config.js (hunk 4 above)
Comment (abarmat) [reply truffle-config.js]: 👍

File: test/LANDRegistry.js (hunk 3 above)
Comment (abarmat) [reply test/LANDRegistry.js]: 👍

File: contracts/land/LANDStorage.sol (hunk 1 above)
Comment (abarmat) [reply contracts/land/LANDStorage.sol]: I think it's a good idea, will take the first option. We can get when it was changed from the commit log.

**Pull request merged**